.gitlab-ci.yml

```diff
@@ -21,21 +21,31 @@ generate-scripts:
       - install_script_agent6.sh
       - install_script_agent7.sh
 
+unit_tests:
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64:$DATADOG_AGENT_BUILDIMAGES
+  tags: [arch:amd64]
+  stage: test
+  dependencies: ["generate-scripts"]
+  script:
+    - sudo apt-get install -y shunit2 yamllint
```

Comment:
Ideally this should be done in the image, but if we want to do it here then we probably should `apt-get update` before to avoid using a potentially outdated package